modules/ROOT/pages/release-notes.adoc

```diff
@@ -9,6 +9,23 @@ The 2022.2 is in development.
 
 == New values added
 
+=== Bonita now embeds the bpmn-visualization library to improve process monitoring for administrators and users 
+
+Bonitasoft has launched the Process Analytics initiative two years ago, to create a new, Open Source means to monitor process execution using the bpmn diagram generated by Bonita Studio. +
+The deliverable of this project is called the _bpmn-visualization library_ . You can find a nice array of examples of what it can do in its https://cdn.statically.io/gh/process-analytics/bpmn-visualization-examples/v0.25.0/examples/index.html[demo webiste], and you can even contribute to this initiative in its https://github.com/process-analytics/bpmn-visualization-js[Github repository]. +
```

Comment:
```suggestion
The deliverable of this project is called the _bpmn-visualization library_ . You can find a nice array of examples of what it can do in its https://cdn.statically.io/gh/process-analytics/bpmn-visualization-examples/v0.25.0/examples/index.html[demo web site], and you can even contribute to this initiative in its https://github.com/process-analytics/bpmn-visualization-js[Github repository]. +
```